include/flatbuffers/flatbuffers.h

```diff
@@ -697,6 +697,24 @@ class vector_downward {
     }
   }
 
+  void reset(size_t initial_size = 1024, Allocator *allocator = nullptr,
+             bool own_allocator = false) {
```

Comment:
I'm taking a look at this now, it might be a vestige of some older impl details with an older iteration of the allocator API.